build/Versions.props

```diff
@@ -1,9 +1,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <VisualStudioVersion>16.3.0</VisualStudioVersion>
-    <ProjectSystemVersion>3.3.0</ProjectSystemVersion>
-    <VersionBase Condition="'$(UseVisualStudioVersion)' == 'true'">$(VisualStudioVersion)</VersionBase>
```

Comment:
Can we remove this UseVisualStudioVersion throughout the code base?

Reply:
`UseVisualStudioVersion` is no longer used